JobConfig/primary/IPAMuminusMichel.fcl

```diff
@@ -6,16 +6,57 @@
 #
 #include "Production/JobConfig/primary/IPAStopParticle.fcl"
 
-physics.filters.PrimaryFilter.MinimumPartMom : 30.0 // MeV/c TODO - tweak this parameter to get more DetSteps
+physics.filters.PrimaryFilter.MinimumPartMom : 1.0
 physics.producers.FindMCPrimary.PrimaryProcess : "mu2eMuonDecayAtRest"
 
-physics.producers.generate : {
-  module_type : Pileup
-  verbosity : 0
-  inputSimParticles: IPAMuminusStopResampler
-  captureProducts : []
-  decayProducts : [ @local::Pileup.dioGenTool ]
-  stoppingTargetMaterial : "IPA"
+CompositeGenerator: {
+  module_type: CompositeMaterialGenerator
+  tag: IPAMuminusStopResampler
+  process: "mu2eMuonDecayAtRest"
```

Comment:
I would also decorate "process" with an adjective.  It's a processCode .

Reply:
The latest push in https://github.com/Mu2e/Offline/pull/1290 exchanges `process` for `processCode`.

Reply:
Thanks - I see the matching change here.